Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -1613,19 +1613,21 @@ public void cleverTapSetLocation(){
     }
 
     @JavascriptInterface
-    public void showDialer(String phoneNum) {
+    public void showDialer(String phoneNum, boolean call) {
```

Comment:
`    public void showDialer(String phoneNum, boolean call) {
        Intent intent = new Intent();
        phoneNumber = "tel:" + phoneNum;
        if (call) {
            if (ContextCompat.checkSelfPermission(activity, Manifest.permission.CALL_PHONE) != PackageManager.PERMISSION_GRANTED) {
                ActivityCompat.requestPermissions(activity, new String[]{Manifest.permission.CALL_PHONE}, REQUEST_CALL);
                return;
            }
        }
        intent.setAction( call ? Intent.ACTION_CALL : Intent.ACTION_DIAL);
        intent.setData(Uri.parse(phoneNumber));
        activity.startActivity(intent);
    }`
    
This is what I was telling, Can you please check this?